src/Game.jsx

```diff
@@ -1,5 +1,7 @@
 import { useEffect, useState } from 'react';
 import he from 'he';
+import Skeleton from 'react-loading-skeleton';
+import 'react-loading-skeleton/dist/skeleton.css';
```

Comment:
Do you need this css import in all the `jsx` files where you plan to use the loading skeleton?